rpc-server/src/dispatchers/validator.rs

```diff
@@ -31,18 +32,19 @@ impl ValidatorInterface for ValidatorDispatcher {
     type Error = Error;
 
     async fn get_address(&mut self) -> RPCResult<Address, (), Self::Error> {
-        Ok(self.validator.validator_address.read().clone().into())
+        Ok(self.validator.read().validator_address.clone().into())
     }
 
+    // TODO: why do we give out secret keys via RPC?
```

Comment:
Action this?

Reply:
Added an issue: #3270.

Theoretically a breaking change so I didn't want to do it here.